.github/workflows/test.yml

```diff
@@ -2,6 +2,8 @@ name: Test sdk
 
 on:
   push:
+  schedule:
```

Comment:
as @rawdaGastan mentioned we don't need unit tests in our nightly workflows, the rest is fine